### PR TITLE
changed CSS properties to place bootstrap menus wrt to screens

### PR DESCRIPTION
**Pull request by ashmichheda**

### CODE REVIEW

File: vspace/src/main/webapp/WEB-INF/views/exhibition/space.html [hunk 1]
```diff
@@ -20,71 +20,84 @@
 
 .Info_cz_Class:hover .tooltiptext {
     visibility: visible;
-}
-
+} */
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/exhibition/space.html:23]: remove comments if not needed

File: vspace/src/main/webapp/WEB-INF/views/exhibition/space.html [hunk 2]
```diff
@@ -20,71 +20,84 @@
...
+            if(window.matchMedia(maxWidthSmallScreens).matches){
+                $("#mySidenav").css("width", "170px");
+                $("#mySidenav").css("height", "260px");
+                $("#mySidenav").css("left", "200px");
+		        $("#mySidenav").css("right", "200px");
+            }
+			/*
+			 Condition for big screens
+			*/
+			else if (window.matchMedia(maxWidth).matches){
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/exhibition/space.html:97]: indentation

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
It's actually almost less usable now. The links are partially not displayed on my phone. The menu should open above the image like the bootstrap menus do, big enough to select them with a finger.
image

**Comment by ashmichheda** [comment]
I went through an example of bootstrap menu. By default, the menu appears extended when the screen is maximized and they collapse when the screen is minimized.
But in our application, the menu is already collapsed when maximized and when the screen is minimized we still want the menu to be collapsed. 
Only on small screens, it should show above the image. I am not sure.

I am going through the below example.

[Boostrap NavBar](https://getbootstrap.com/docs/3.4/examples/navbar/)

Is this the right one to refer as an example?

**Comment by ashmichheda** [comment]
Should the menu be along with the sign out and dashboard icons for small screens?

**Pull request closed without merging**